packages/ui-components/src/components/button.tsx

```diff
@@ -19,6 +20,10 @@ export interface IButtonProps
    * Whether this button should use small styles.
    */
   small?: boolean;
+  /**
+   * Button content
+   */
+  value?: string;
```

Comment:
This is unused. The content should be automatically available through the `children` prop (see the [documentation](https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children) for more info).

```suggestion
```

Reply:
Yes it is available but here i used toolkit button component as per interface its `value?: string;` and in the button component as per interface its `value?: string | ReadonlyArray<string> | number | undefined;`


So for uniformity i added `value?: string`.